crates/aptos-protos/proto/aptos/datastream/v1/datastream.proto

```diff
@@ -5,31 +5,43 @@ syntax = "proto3";
 
 package aptos.datastream.v1;
 
+import "aptos/util/timestamp/timestamp.proto";
+
+// Transaction data is transferred via 1 stream with batches until terminated.
+// One stream consists:
+//  StreamStatus: INIT with version x
+//  loop k:
+//    TransactionOutput data(size n)
+//    StreamStatus: BATCH_END with version x + k * n
+
 message TransactionsOutput {
   repeated TransactionOutput transactions  = 1;
 }
 
 message TransactionOutput {
+  // Encoded aptos.proto.v1.Transaction proto data.
   string encoded_proto_data = 1;
   uint64 version = 2;
+  aptos.util.timestamp.Timestamp timestamp = 3;
 }
 
 message StreamStatus {
   enum StatusType {
-    BATCH_START = 0;
+    // Signal for the start of the stream.
+    INIT = 0;
+    // Signal for the end of the batch.
```

Comment:
Do we also need a signal to gracefully close the stream, along with a close reason?

Reply:
We haven't added end version yet (coming later) so there's really no graceful closing of the stream currently.